.mise.toml

```diff
@@ -13,11 +13,6 @@ go = "1.24"
 gofumpt = "latest"
 golangci-lint = "1.64.2"
 jq = "latest"
-node = "20"
```

Comment:
Two things happening here:
- This is installed in the `sdk/nodejs` directory, which means it won't need to be installed for non-node jobs. If I'm being too naïve here, we can move it back, though.
- We don't need `typescript` and `typescript-language-server` because they can be listed in the `package.json`, and the `env` key of the new `.mise.toml` file should set them up on the $PATH.

Reply:
> This is installed in the sdk/nodejs directory, which means it won't need to be installed for non-node jobs. If I'm being too naïve here, we can move it back, though.

Yeh there will be integration tests that try to run node programs or providers.

Reply:
Integration tests will need the various language runtimes. Not opposed to having a `.mise.toml` inside the tests folder though, if that's neater?

Reply:
@Frassle Thanks! I'll move it back. @julienp I think that's probably a nicer solution, I'll add it to my list - just trying to understand the current state of the world rather than fix anything yet 🫠